gradle-plugin/plugin/build.gradle.kts

```diff
@@ -140,7 +140,7 @@ dependencies {
   testRuntimeOnly(libs.junit.jupiter.engine)
   testRuntimeOnly(libs.junit.platform.launcher)
 
-  "functionalTestImplementation"(projects.emergeGradlePlugin.plugin)
+  "functionalTestImplementation"(project(":plugin"))
```

Comment:
This caused the same compilation issue we were seeing before when adding this as a composite build so I just removed the generated accessor.